pkg/model/manifest.go

```diff
@@ -35,7 +35,7 @@ import (
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 	"github.com/okteto/okteto/pkg/model/forward"
 	"github.com/spf13/afero"
-	yaml "gopkg.in/yaml.v2"
```

Comment:
goland insists in removing this, which is redundant, happy to revert it if we prefer so.